keps/sig-node/127-user-namespaces/README.md

```diff
@@ -195,11 +196,72 @@ impact a compromised pod can have on other pods and the node itself.
 
 The following changes will be done to the pod.spec:
 
-- `pod.spec.hostUsers`: bool.
+- `pod.spec.hostUsers`: *bool.
 If true or not present, uses the host user namespace (as today)
 If false, a new userns is created for the pod.
 By default it is set to `true`.
```

Comment:
let's make the default "not set" with the meaning of true

Reply:
/lgtm cancel

Reply:
+1

Reply:
Fixed, thanks!